src/components/phylogeny/PhylogenyView.vue

```diff
@@ -69,12 +69,12 @@
             </div>
           </div>
 
-          <!-- Primary reference phylogeny -->
+          <!-- Primary reference phylogeny
           <Citation
             label="Primary reference phylogeny"
             :object="selectedPhylogeny"
             citation-key="primaryPhylogenyCitation"
-          />
+          /> -->
```

Comment:
Given the description on the PR, am I right this appears to be "confusing"? If so, that fact isn't evident from the code, so I'd suggest adding a comment as to why we're commenting it out for now. Meaning this is a temporary measure, and how do we intend to fix the confusing nature; perhaps refer to an issue to be created on the tracker. Or if this is not temporary, why not remove it altogether?